### ISSUE TITLE
Data request for species petrale sole

### ISSUE THREAD

**Issue by iantaylor-NOAA** [pfmc-assessments/nwfscDiscard#10]
### Identify the common name(s)

petrale sole

### Type of data needed

discard totals/rates and biological data

### Gear grouping

all gears combined

### Areas

two areas:
1-California
2-Oregon/Washington combined

### Length bins for discard compositions

from 12 cm to 62 cm, by 2 cm increments

### Additional information

This request is identical to the already completed #9 submitted by @gertsevv except that the previous request was for annual rates and now we are interested in seasonal rates as provided in 2019 (defined as Season 1 = Nov (of previous year - Feb) and Season 2 = March - October.

Adding the annual rates to the model had a small but non-trivial impact on the model results so although we're leaning toward an annual model we want to see the impact of extending the discard data for the seasonal model.

### Do you you have clearance to access these data with a valid 2023 NDA form with WCGOP?

- [X] I agree that I have clearance

**Comment by iantaylor-NOAA** [comment]
Adding a comment to trigger notification with updated subject line after changing "[replace with your species]" to "petrale sole". Sorry for spacing out on that.

**Comment by andi-stephens-NOAA** [comment]
Done.  Let me know if there's anything that looks off.

//nwcfile/FRAM/Assessment Data/2023 Assessment Cycle/petrale
sole/wcgop/Petrale_SEASONAL_WCGOP_WAOR-CA_AllGear.xlsx

**Comment by iantaylor-NOAA** [comment]
Looks good, thank you @andi-stephens-NOAA!

**Comment by chantelwetzel-noaa** [comment]
The discard totals and rates have been added to the network.  Adding a seasonal filter broke my new column labeling code for the non-catch share file, so I reverted to the older column names and the additional columns that have been provided in the past. I did not fully trust myself to modify these items by hand so I apologize with the slight variance in the column names compared to the initial data summaries.  Let me know if you have any questions or issues.
